### ISSUE TITLE
Extraction Method defaults to Lattice

### ISSUE THREAD

**Issue by mirhl** [tabulapdf/tabula#459]
I mean.. I just discovered this great tool and all.
And from the description wording, it seems Stream is the "safest" method (in my testings it was the better, indeed). 
> The current preview uses the Stream extraction method. If the data is not mapped to the correct cells, try the Lattice method instead.

Things are: Lattice is on the other hand what I get by default. 
Is this expected?

**Comment by jeremybmerrill** [comment]
@mirhl: Yes, this is expected -- sort of. The problem space (things that appear to humans as tables, laid out in PDFs) is very large, so there are two algorithms to cover different pieces of the space. 

Lattice uses ruling lines, which are often not present at all in the PDF. Stream uses only the positions of the characters, which are, obviously, per se present. We have a heuristic that tries to guess which algorithm to use. It's not perfect, but it's better than nothing: you're getting cases where the heuristic thinks the Lattice method will work better, but is wrong. 

The heuristic is difficult to improve because we don't, per se, know what the output is supposed to look like. (If we did, we'd just output that...)

**Comment by mirhl** [comment]
Thanks for the explanation. 
The tooltip wording could be improved anyway imo.

**Comment by jazzido** [comment]
Thanks, @mirhl. 
We're open to suggestions for improving the wording.

**Comment by jeremybmerrill** [comment]
@mirhl That is useful feedback! (Just because I'm dense; which tooltip are you referring to?)

**Comment by mirhl** [comment]
The one I quoted!
I mean, the current preview *wasn't* using Stream method. :s

**Comment by jeremybmerrill** [comment]
Lol, did we just hardcode that in? Oopsies.
